src/core/Utils.js

```diff
@@ -176,23 +176,46 @@ function getArrayBuffer (file) {
 
 function getFileType (file) {
   const emptyFileType = ['', '']
-  if (file.type) {
-    return Promise.resolve(file.type.split('/'))
+  const extensionsToMime = {
+    'md': 'text/markdown',
+    'markdown': 'text/markdown',
+    'mp4': 'video/mp4',
+    'mp3': 'audio/mp3'
   }
```

Comment:
We should probably have one unified list of extensions/file-types? I need to search it by extension, and the one used in `getFileTypeExtension` is the other way around. But I guess we can just use one list and two different methods, `getFileTypeExtension` and `getExtensionByFileType`, looping through it with Object.keys().forEach?